frontend/src/old-pages/Configure/Storage.tsx

```diff
@@ -704,7 +704,7 @@ function Storage() {
               <Select
                 disabled={editing}
                 placeholder="Please Select a Filesystem Type"
-                selectedOption={storageType && itemToDisplayIconOption(findFirst(storageTypes, (s: any) => {return s[0] === storageType}))}
+                selectedOption={storageType && itemToDisplayIconOption(findFirst(storageTypes, s => s[0] === storageType) || ['', '', ''])}
```

Comment:
issue(non-blocking): by using `['', '' ,'']` as default, wouldn't we pass a possibly non-existing item to `selectedOption`? Would that cause any glitch?

Reply:
Ah I think this is probably better resolved by using `!` (which I just learned about) stating that we will have a match (which fixes the type error)